dev-tools/module_committers

```diff
@@ -0,0 +1,40 @@
+#!/bin/bash
+
+#
+# Finds modules matching given string and prints out their recent top committers.
+#
+# Usage:
+# module_devs <module name, partial ok>
+#
+
+module_name="$1"
+if [ -z "$module_name" ]; then
+    echo "Usage: module_devs <module name>"
+    exit 1
+fi
+
+beats_base_dir=$(cd $(dirname $BASH_SOURCE)/..; pwd)
+
+beat_dirs=$(find $beats_base_dir -type d -iname \*beat -maxdepth 2 | grep -v libbeat)
```

Comment:
I think you can wrap some lines into functions (find_beats_dirs, find_module_dirs, filter_module_name, etc.)